Sources/Visualization/Microsoft.Psi.Visualization.Windows/ViewModels/PartitionViewModel.cs

```diff
@@ -209,7 +209,7 @@ public string AuxiliaryInfo
         /// </summary>
         [DisplayName("Stream Reader Type")]
         [Description("The type of stream reader used by the partition.")]
-        public string StreamReaderTypeName => TypeSpec.Simplify(this.partition.StreamReaderTypeName);
+        public string StreamReaderTypeName => this.partition.StreamReaderTypeName;
```

Comment:
What if instead line 413 changed to `TypeResolutionHelper.GetVerifiedType(this.partition.StreamReaderTypeName),`
Keeping the property here simplified for display in the UI, but using the underlying `partition.StreamReaderTypeName` to create the `StreamSouce`?

Reply:
Good suggestion. Also needed to change line 102 in StreamTreeNode.cs. And renamed the property to `StreamReaderTypeDisplayName` just to eliminate any confusion.